src/PsfexPsf.cc

```diff
@@ -110,6 +110,11 @@ PsfexPsf::resized(int width, int height) const {
     throw LSST_EXCEPT(pex::exceptions::LogicError, "Not Implemented");
 }
 
+// Get the dimensionality of the context data
```

Comment:
No docstrings on function definitions, it's already on the declaration in the header file.